pdns/lua-record.cc

```diff
@@ -128,7 +130,10 @@ class IsUpOracle
 
       if(!status) {
         g_log<<Logger::Info<<"LUA record monitoring declaring "<<remstring<<" UP for URL "<<cd.url<<"!"<<endl;
+        g_log<<Logger::Info<<"LUA record monitoring declaring "<<remstring<<" WEIGHT is "<<content<<"!"<<endl;
```

Comment:
maybe the weight can go into the existing message

Reply:
sure

Reply:
merged the log, added a try catch just in case the content is not a weight(int)